pkg/lib/serverTime.js

```diff
@@ -69,8 +71,10 @@ export function ServerTime() {
 
     const timedate1_service = service.proxy("dbus-org.freedesktop.timedate1.service");
     const timesyncd_service = service.proxy("systemd-timesyncd.service");
+    const chronyd_service = service.proxy("chronyd.service");
```

Comment:
> Hardcoded chronyd.service -- the Python script also accepts `chrony.service`. Is that relevant on some distros?

Debian et al calls this unit chrony.service and has a alias named chronyd.service.  Fedora et al call this just chronyd.service.  Thus chronyd.service works on all our supported OSes.

> If so, then the py script should just return the name, you use `.startswith("chrony")`, and save the unit name. 

The Python script hasn't been run yet when this code is executed. I don't think we should run it on every page load since chronyd.service works just fine.

> Otherwise, please drop "chrony.service" from the python script.

Debian et al have the alias, but they put "chrony.service" into the ntp-units files, so we need to recognize that.

> But making that return the actual unit name might be a cleaner design anyway? "unsupported" has no special meaning, it could just return an empty string if it didn't find any supported unit name.

Maybe... then JavaScript has to treat chronyd and chrony as identical. I don't think it makes a big enough difference.

Reply:
Ack, thanks for the explanation! Sounds fine. However, that could use a comment then to explain the dissonance between this hardcoded unit name and the two names in the python script, so that our future selves don't wonder about that. (PR review comments are too hard to find).

Reply:
> Ack, thanks for the explanation! Sounds fine. However, that could use a comment then to explain the dissonance between this hardcoded unit name and the two names in the python script, so that our future selves don't wonder about that. (PR review comments are too hard to find).

Yeah... I changed my mind.  Originally, get-timesyncd-backend would do more work to see whether Cockpit actually supports the backend. Specifically, it would check whether /etc/chrony.d/sources exists. And it did that purely because it was less code to do it in Python...